+++
title = "The elusive long-read fungal sequence: how to minION like a pro (when you’ve never done it before)"
authors = ["Ms. Lauren Huth - University of Southern Queensland Institute for Life Sciences and the Environment"]
date = "2020-12-01"
time_start = 2020-12-01T10:00:00
time_end = 2020-12-01T11:30:00

# Abstract
abstract = "<img src = '/img/seminars/lauren-huth.jpg' style = 'padding:40px; float:right'> The rapid development of microbial genome sequencing methods in the last decade has revolutionized whole genome sequencing and has become the standard for many molecular typing applications in research. Largely driven by the development of high-throughput, low cost, second generation sequencing methods, the industry has become familiar with Illumina as the most widely used sequencing platform for microbial genomics. However, it has become clear that short-read sequencing has significant limitations, in particular, the inability to fully reconstruct genomic structures of interest. Oxford Nanopore Technologies offer a solution to this employing a single-molecule, long-read sequencing platform called minION often generating reads with a median length of 8-10kb and as long as 100kb. In-house sequencing can be a daunting task, even for those experienced in the process. In addition to this, fungi can be particularly difficult to prepare compared to other samples. This talk will provide a beginner’s overview of Oxford Nanopore Technology and their platforms including the minION and flongle and what is required for a successful sequencing run. In particular, this seminar will focus on high molecular weight DNA extraction of fungi, quantification, purification, purity ratio checks, short read eliminators and library preparation using the Genomic DNA by Ligation sequencing kits. There will be an overview of costs as well as some tips and tricks to get the most out of your sequencing run. The rapid development of microbial genome sequencing methods in the last decade has revolutionized whole genome sequencing and has become the standard for many molecular typing applications in research. Largely driven by the development of high-throughput, low cost, second generation sequencing methods, the industry has become familiar with Illumina as the most widely used sequencing platform for microbial genomics. However, it has become clear that short-read sequencing has significant limitations, in particular, the inability to fully reconstruct genomic structures of interest. Oxford Nanopore Technologies offer a solution to this employing a single-molecule, long-read sequencing platform called minION often generating reads with a median length of 8-10kb and as long as 100kb. In-house sequencing can be a daunting task, even for those experienced in the process. In addition to this, fungi can be particularly difficult to prepare compared to other samples. This talk will provide a beginner’s overview of Oxford Nanopore Technology and their platforms including the minION and flongle and what is required for a successful sequencing run. In particular, this seminar will focus on high molecular weight DNA extraction of fungi, quantification, purification, purity ratio checks, short read eliminators and library preparation using the Genomic DNA by Ligation sequencing kits. There will be an overview of costs as well as some tips and tricks to get the most out of your sequencing run."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected seminar? (true/false)
selected = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["teaching-notes"]

# Links (optional).
#url_slides = ""
#url_video = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
url_custom = [{name = "Zoom Link", url = "https://usq.zoom.us/j/85444037707"}]


# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = "My caption :smile:"
+++
